CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - oe_verify_attestation_certificate_with_evidence() has been deprecated because it has been deemed insufficient for security. Use the new, experimental oe_verify_attestation_certificate_with_evidence_v2() instead to generate a self-signed certificate for use in the TLS handshaking process.
 - In/out parameters in EDL now have the default count equals to one if the `count` attribute is not used.
 - Improved attestation evidence verification performance.
+- Retrieved V3 DCAP collateral with the CRL in DER format by default.
```

Comment:
You can combine this with line 26. For example,

"Add the support for SGX quote verification collateral version 3 with the CRL in DER format by default..."